template/runtime/src/precompiles.rs

```diff
@@ -42,12 +34,14 @@ where
 			// Non-Frontier specific nor Ethereum precompiles :
```

Comment:
Looks like there's some missing precompiles...

https://www.evm.codes/precompiled

We should have a separate ticket for these; probably `blake2f` is the most important.